src/blog/2021/05/welcome-sam.md

```diff
@@ -3,6 +3,9 @@ title: Welcome Sam
 description: Welcoming Sam Machin to FlowForge Inc.
 date: 2021-05-17
 authors: ["nick-oleary"]
+tags:
```

Comment:
Should we either add `posts` or just remove the post?

As post is a catch-all it seems like this one was forgotten?

Reply:
yep, just missed by accident